dev/com.ibm.ws.ejbcontainer.core/src/com/ibm/ejs/container/util/ExceptionUtil.java

```diff
@@ -273,25 +231,19 @@ static public Throwable findCause(Throwable throwable)
                    cause instanceof CSIException ||
                    cause instanceof InjectionException || // d436080
```

Comment:
In the method below we added ManagedObjectException to this list ... should it have been added here too?

Reply:
The comment says it's looking for a non websphere specific exception and ManagedObjectException is our class.